ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
@rahmanusta I'll add the versions to my personal branch which includes your patch + minor additions. But please note that we also have to implement the packaging as discussed below and that we still have the failing tests. So perhaps it would make sense to create a new clean PR with the structure discussed below and check whether this gets the tests green?